circle.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
       I="image-$(date +%j).tar"; mkdir -p ~/docker; rm ~/docker/*; docker save app:build > ~/docker/$I; ls -l ~/docker
 test:
   override:
-    - docker run -d --net=host app:build; sleep 10
+    - docker run -d --net=host app:build START; sleep 10
```

Comment:
Is this just to be explicit? Since we have `CMD ["START"]` in the `Dockerfile` already?